x-pack/solutions/observability/plugins/apm/docs/openapi/apm/bundled.json

```diff
@@ -1334,30 +1380,32 @@
         "properties": {
           "@timestamp": {
             "type": "string",
-            "description": "Timestamp"
+            "description": "The date and time of the annotation. It must be in ISO 8601 format."
           },
           "service": {
             "type": "object",
-            "description": "Service",
+            "description": "The service that identifoes the configuration to create or update.",
```

Comment:
```suggestion
            "description": "The service that identifies the configuration to create or update.",
```

Reply:
Thanks! I copied these changes into the source files.